3.-Web-api-call-Microsoft-graph-for-personal-accounts/README.md

```diff
@@ -267,194 +224,120 @@ Explore the sample by signing in into the TodoList client, adding items to the T
 
-Calling a downstream API involves getting a token for this Web API. Acquiring a token is achieved by using MSAL.NET.
+This section, here, is only about the differences in the code to let the Web API call the Microsoft Graph with Microsoft personal accounts
 
-Reference the `Microsoft.Identity.Client` NuGet package from the TodoListService project.
+### Change to the Web API (TodoListService)
 
-Add a reference to the `Microsoft.Identity.Web` library. It contains reusable code that you can use in your Web APIs (and web apps)
-
-### Modify the startup.cs file to add a token received by the Web API to the MSAL.NET cache
+### Modify the startup.cs file so that access tokens for Microsoft personal accounts can be refreshed
 
 Update `Startup.cs` file:
 
-- Add a using for `Microsoft.Identity.Client`
-
-- In the `ConfigureServices` method, replace:
-
-  ```CSharp
-  services.AddAuthentication(AzureADDefaults.BearerAuthenticationScheme)
-          .AddAzureADBearer(options => Configuration.Bind("AzureAd", options));
-   ```
-
-  by
+- In the `ConfigureServices` method, in the call to `AddProtectedApiCallsWebApis`, you need to add the `offline_access` scope. For this, replace:
 
-  ```csharp
-  services.AddProtectWebApiWithMicrosoftIdentityPlatformV2(Configuration)
-          .AddProtectedApiCallsWebApis(Configuration, new string[] { "user.read" })
-          .AddInMemoryTokenCaches();
-  ```
-
-  `AddProtectWebApiWithMicrosoftIdentityPlatformV2` does the following:
-  - add the **Jwt**BearerAuthenticationScheme (Note the replacement of BearerAuthenticationScheme by **Jwt**BearerAuthenticationScheme)
-  - set the authority to be the Microsoft identity platform v2.0 identity
-  - sets the audiences to validate
-  - register an issuer validator that accepts issuers to be in the Microsoft identity platform clouds.
-  - 
+```CSharp
+public class Startup
+{
+  ...
+  // This method gets called by the runtime. Use this method to add services to the container.
+   public void ConfigureServices(IServiceCollection services)
+   {
+    services.AddProtectWebApiWithMicrosoftIdentityPlatformV2(Configuration)
+            .AddProtectedApiCallsWebApis(Configuration, new string[] { "user.read" })
+            .AddInMemoryTokenCaches();
+    services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+   }
+   ...
+}
+```
 
-  Here is an idea of the code: 
+by
 
-  ```csharp
-  services.AddAuthentication(AzureADDefaults.JwtBearerAuthenticationScheme)
-          .AddAzureADBearer(options => configuration.Bind("AzureAd", options));
+```csharp
+public class Startup
+{
+  ...
+  public void ConfigureServices(IServiceCollection services)
+  {
+      services.AddProtectWebApiWithMicrosoftIdentityPlatformV2(Configuration)
```

Comment:
>V2 [](start = 60, length = 2)

nit: do we want to have "v2" here or something else or nothing? i don't think it's needed because Microsoft Identity Platform is already there

Reply:
This is a question for the Microsoft.Identity.Web library. It will go through API review.